message/message.go

```diff
@@ -13,14 +14,14 @@ type Message struct {
 	Visibility          string  `json:"visibility"`
 	ExternalRef         *string `json:"external_ref,omitempty"`
 	// @todo: Legacy fields. Need to deprecate over time.
-	Standards           []string `json:"standards,omitempty"`
-	Audits              *[]Audit `json:"audits,omitempty"`
+	Standards []string `json:"standards,omitempty"`
+	Audits    *[]Audit `json:"audits,omitempty"`
 }
 
 // @todo: Legacy. Needs to be deprecated over time.
 type Audit struct {
-	Type    string        `json:"type"`
-	Options *AuditOption  `json:"options"`
+	Type    string       `json:"type"`
+	Options *AuditOption `json:"options"`
```

Comment:
We should also add `omitempty` to `Options`.

Reply:
Yup!

Reply:
can do